src/Uno.ReferenceImplComparer/Program.cs

```diff
@@ -46,13 +46,41 @@ static int Main(string[] args)
 			return hasErrors ? 1 : 0;
 		}
 
+		private static bool IsAccessible(MethodDefinition method)
+		{
+			// https://github.com/jbevain/cecil/blob/56d4409b8a0165830565c6e3f96f41bead2c418b/Mono.Cecil/MethodAttributes.cs#L22-L24
+			return method.IsPublic || method.IsFamily || method.IsFamilyOrAssembly;
+		}
+
+		private static bool IsAccessible(FieldDefinition field)
+		{
+			// https://github.com/jbevain/cecil/blob/56d4409b8a0165830565c6e3f96f41bead2c418b/Mono.Cecil/FieldAttributes.cs#L22-L24
+			return field.IsPublic || field.IsFamily || field.IsFamilyOrAssembly;
+		}
+
+		private static bool IsAccessible(TypeDefinition type)
+		{
+			// https://github.com/jbevain/cecil/blob/56d4409b8a0165830565c6e3f96f41bead2c418b/Mono.Cecil/TypeAttributes.cs#L21-L26
+			return type.IsPublic || type.IsNestedPublic || type.IsNestedFamily || type.IsNestedFamilyOrAssembly;
+		}
+
+		private static bool IsAccessible(PropertyDefinition property)
+		{
+			return property.GetMethod is not null && IsAccessible(property.GetMethod);
```

Comment:
Should this also check for setters?

Reply:
Yeah I think it should, but I kept it the same for simplicity for now

Reply:
It should even check for the property accessibility itself.

Reply:
Somehow, PropertyDefinition doesn't seem to have `IsPublic` property and its friends. So keeping the original behavior for now to check the getter only which is 99.9999% has the same accessibility as the property itself.

Reply:
> I think it should even check for the property accessibility itself (not getters or setters). However, I never saw code where the getter doesn't match the property accessibility.

I take that back. We should check both.